web_src/css/admin.css

```diff
@@ -8,6 +8,7 @@
   padding: 0;
   font-size: 13px;
   overflow-x: auto;
+  border-radius: 0 0 var(--border-radius) var(--border-radius);
```

Comment:
Why we need this? I can see there is no bottom radius for other elements.

![image](https://user-images.githubusercontent.com/2114189/235362544-f60bc192-bd79-48dd-a428-0c875efb7bd0.png)

Reply:
I guess I can try fixing it on a more "global" level. I think there should be a border-radius on such boxes, but the issue is that content with background may break out of the border radius area, so it may not work on every box.

Reply:
Reverted, I will defer this to another time.